packages/next/src/utils.ts

```diff
@@ -2377,6 +2379,45 @@ interface EdgeFunctionMatcher {
   originalSource?: string;
 }
 
+const vercelFunctionRegionsVar = process.env.VERCEL_FUNCTION_REGIONS;
+let vercelFunctionRegions: string[] | undefined;
+if (vercelFunctionRegionsVar) {
+  vercelFunctionRegions = vercelFunctionRegionsVar.split(',');
+}
+
+/**
+ * Normalizes the regions config that comes from the Next.js edge functions manifest.
+ * Ensures that config like `home` and `global` are converted to the corresponding Vercel region config.
+ * In the future we'll want to make `home` and `global` part of the Build Output API.
+ * - `home` refers to the regions set in vercel.json or on the Vercel dashboard project config.
+ * - `global` refers to all regions.
+ */
+function normalizeRegions(regions: Regions): string[] {
+  const newRegions: string[] = [];
+  for (const region of regions) {
```

Comment:
It appears as if `regions` could be a `string` or `string[]`, so in the event it's a string (e.g. `"home"`), consider arrayifying it:

```suggestion
  if (!Array.isArray(regions)) {
    regions = [regions];
  }

    for (const region of regions) {
```

Reply:
Thank you, super good catch! Got bit by the "string can be iterated" with TypeScript once again 🤦